virttest/qemu_vm.py

```diff
@@ -3544,6 +3544,21 @@ def get_spice_var(self, spice_var):
         """
         return self.spice_options.get(spice_var, None)
 
+    def _get_bus(self, params, dtype=None, pcie=False):
+        """
+        Deal with different buses for multi-arch
+        """
+        if self.params.get("machine_type").startswith('s390'):
+            return self._get_ccw_bus()
+        else:
+            return self._get_pci_bus(params, 'nic', pcie)
```

Comment:
`_get_pci_bus` not only returns the nic bus.

Reply:
updated